labs/lab-1-html-and-css/style.css

```diff
@@ -0,0 +1,151 @@
+body {
+	background-color: #f9fafc;
+	font-family: 'Open Sans' , sans-serif ;
+	color: #69696a;
+
+
+}
+
+/*nav */
```

Comment:
This is a Bad Comment™! I can tell it's the nav section by the way it says `nav {`  and `.nav {` right there!